AdobeStockAsset/etc/db_schema.xml

```diff
@@ -9,14 +9,10 @@
     <table name="adobe_stock_asset" resource="default" engine="innodb" comment="Adobe Stock Asset">
         <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
         <column xsi:type="int" name="media_type_id" padding="10" unsigned="true" nullable="true" identity="false" comment="Media Type ID"/>
```

Comment:
Please remove `media_type_id` filed. Looks like it's duplicate to `content_type` (`content_type` is more detailed)